### PR TITLE
feat: New template for second factor authentication email (SQSERVICES-1307)

### PR DESCRIPTION
**Pull request by battermann**
[SQSERVICES-1307](https://wearezeta.atlassian.net/browse/SQSERVICES-1307)
https://wearezeta.atlassian.net/browse/SQSERVICES-1307

----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [x] contains a reference JIRA issue number like `SQPIT-764`
  - [x] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [x] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

### Issues

Added a new template for a second factor authentication email containing a verification code and the related action (placeholder).

### Solutions

Added a new template.

### Dependencies (Optional)

No dependencies. But is needed for https://github.com/wireapp/wire-server/pull/2142

### Testing

Manually viewed the HTML locally.

### CODE REVIEW

**Comment by battermann** [comment]
There were some issues with outdated/incompatible node/library version, that I could fix with the help of @tlebon fortunately. Therefore, the changes in the deployment pipeline and in `gulpfile.js`.

There still seems to be an issue with crowdin. Possibly an expired access token. @Astrid-Wire maybe you can help out here?

File: src/pages/en/user/email/second-factor-verification.html [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+subject: "${code} is your ${brand} verification code"
```
Comment (fisx) [inline src/pages/en/user/email/second-factor-verification.html:2]:
```suggestion
subject: "your ${brand} verification code is ${code}"
```

File: src/pages/en/user/email/second-factor-verification.html [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+---
+
+<container>
+  <row>
+    <columns small="12">
+      <h1>Verify ${action}</h1>
+      <p><strong>${email}</strong> was used to ${action}. Enter this code to verify your email and ${action}.</p>
```
Comment (fisx) [inline src/pages/en/user/email/second-factor-verification.html:9]: It seems like we need to translate the action as well.  I'm not sure we have a solution for that?  @Astrid-Wire?

Also, I'm pretty sure you can't use action in both those grammatical contexts in all languages and not break at least one or the other.  Maybe just:

```suggestion
      <p><strong>${email}</strong> was used for action ${action}. Enter this code to confirm this in the app.</p>
```

**Review by fisx — COMMENTED** [review]
A few comments on the template.  I trust you guys with the js dependency hell management (even though I'm in awe facing the size of `yarn.lock`).

**Pull request merged**